src/components/CanvasWrapper.tsx

```diff
@@ -616,12 +650,32 @@ export default function CanvasWrapper(inputProps: CanvasWrapperProps): ReactElem
       </span>
     )
   );
+  const labelData: LabelData | undefined = labels[props.annotationState.currentLabelIdx ?? 0];
+  const shouldShowReuseValueHotkey = labelData?.options.type === LabelType.INTEGER && labelData?.options.autoIncrement;
 
   return (
     <CanvasContainer ref={containerRef} $annotationModeEnabled={props.annotationState.isAnnotationModeEnabled}>
-      {props.annotationState.isAnnotationModeEnabled && (
-        <AnnotationModeContainer>Annotation editing in progress...</AnnotationModeContainer>
-      )}
+      {
+        // TODO: Fade out annotation mode modal if mouse approaches top left corner?
+        props.annotationState.isAnnotationModeEnabled && (
+          <AnnotationModeContainer>
+            <span style={{ marginLeft: "2px" }}>
+              <b>Annotation editing in progress...</b>
+            </span>
+            <FlexColumn $gap={6}>
+              <FlexRowAlignCenter $gap={6}>
+                <HotkeyText>Shift</HotkeyText> hold to select range
+              </FlexRowAlignCenter>
+              {shouldShowReuseValueHotkey && (
+                <FlexRowAlignCenter $gap={6}>
+                  <HotkeyText>Ctrl</HotkeyText>
+                  hold to reuse last value
+                </FlexRowAlignCenter>
+              )}
+            </FlexColumn>
+          </AnnotationModeContainer>
+        )
+      }
```

Comment:
Changed the popup in the corner to include shortcut keys. The visible keys are context-dependent, so the `Ctrl` hotkey is only shown if the current label automatically changes values (e.g. auto-incrementing integer labels)

![image](https://github.com/user-attachments/assets/2ccfb22b-54fc-4910-aae3-c5f1e852f40e)